app/uk/gov/hmrc/gform/auth/models/RecoverAuthResult.scala

```diff
@@ -48,12 +50,24 @@ object RecoverAuthResult {
   def recoverNoActiveSession(
     request: Request[AnyContent],
     appConfig: AppConfig
-  ): PartialFunction[Throwable, AuthResult] = { case _: NoActiveSession =>
-    logger.debug("No Active Session")
-    val continueUrl = java.net.URLEncoder.encode(appConfig.`gform-frontend-base-url` + request.uri, "UTF-8")
-    val ggLoginUrl = appConfig.`government-gateway-sign-in-url`
-    val url = s"$ggLoginUrl?continue=$continueUrl"
-    AuthRedirectFlashingFormName(url)
+  ): PartialFunction[Throwable, AuthResult] = {
+    case _: NoActiveSession if request.method == "POST" =>
```

Comment:
Let's make this the same for all requests, no need to handle only POST differently.